mlflow/java/client/src/main/java/org/mlflow/tracking/MlflowClient.java

```diff
@@ -133,6 +133,24 @@ public long createExperiment(String experimentName) {
     return mapper.toCreateExperimentResponse(ojson).getExperimentId();
   }
 
+  /** Mark an experiment and associated runs, params, metrics, etc for deletion. */
+  public void deleteExperiment(long experimentId) {
```

Comment:
Also added delete & restore experiment to Java Tracking API (already exists in Python API).